Framework/Algorithms/src/SortXAxis2.cpp

```diff
@@ -44,23 +49,24 @@ void SortXAxis::exec() {
   MatrixWorkspace_sptr outputWorkspace = inputWorkspace->clone();
 
   // Check if it is a valid histogram here
-  bool isAProperHistogram = determineIfHistogramIsValid(inputWorkspace);
+  bool isAProperHistogram = determineIfHistogramIsValid(inputWorkspace.get());
 
   // Define everything you can outside of the for loop
   // Assume that all spec are the same size
   const auto sizeOfX = inputWorkspace->x(0).size();
 
-  PARALLEL_FOR_IF(Kernel::threadSafe(*inputWorkspace, *outputWorkspace))
+  PARALLEL_FOR_IF(
+      Kernel::threadSafe(*inputWorkspace.get(), *outputWorkspace.get()))
```

Comment:
Was the original `Kernel::threadSafe(*inputWorkspace, *outputWorkspace)` not okay?

Reply:
I'd changed them to fit with the other required .get() in the code around it. It wasn't necessary, but it made it easier to read, at least for me.

Reply:
Changed to fit the rest of the code around it.